Documentation/GeneralConventions/GuidesXml.rst

```diff
@@ -0,0 +1,199 @@
+..  _guides-xml:
+
+================================
+:file:`Documentation/guides.xml`
+================================
+
+This XML file contains meta information and configuration used during rendering
+of a manual.
+
+Example:
+
+..  literalinclude:: _guides-simple.xml
+    :capition: Documentation/guides.xml
+
+..  hint::
+    If you are migrating from the legacy Sphinx-based rendering and still have
+    a :file:`Documentation/Settings.cfg` you can use an automatic migration
+    tool to :ref:`migrate the settings.cfg into a guides.xml <migrate_guides_xml>
+
+.. _settings-guides:
+
+The `<guides>` tag
+==================
+
+It is used for general configuration during parsing and rendering.
+
+.. _settings-guides-project:
+
+The `<project>` tag
+===================
+
+This tag can contain the following meta information:
+
+Project title
+-------------
+
+The *project* property contains the title of the project and is displayed in the
+left sidebar (desktop) or the top (mobile) of the theme and in the title meta
+tag.
+
+For TYPO3 extensions we suggest to use the extension name in a readable
+format:
+
+*   Import / Export tool
+*   News
+*   Mask
+
+Common values are in the official TYPO3 manuals
+
+#. `<Topic> Guide`, e.g. "Frontend Localization Guide",
+   for collections of articles on a specific topic
+#. `<Topic> Reference`, e.g. "TSconfig Reference",
+   for a complete reference
+#. `<Topic> Tutorial`, e.g. "Editors Tutorial",
+   for collections of tutorials on a specific topic
+
+.. _settings-guides-version-and-release:
+
+Version and release
+-------------------
+
+The properties *version* and *release* both contain the version of the manual
+and mostly correspond to the version of the TYPO3 LTS or TYPO3 extension to
+which the documentation refers.
+
+The version is shown below the title in the theme's release switch and in the
+title meta tag, the release is not shown currently  - but it should be
+kept anyway to satisfy internal requirements.
+
+version
+    The major project version, used as the replacement for \|version\|.
+    For example this may be something like 12.4.
+
+release
+    The full project version, used as the replacement for \|release\|.
+    For example 13.0.0-dev.
+
+If you do not need the separation provided between version and release,
+just set them both to the same value.
+
+.. _settings-guides-copyright:
+
+Copyright
+---------
+
+The *copyright* property contains the copyright claim of the project. It is
+displayed in the footer as "© Copyright <copyright>" and has in most use cases
+of the TYPO3 world the values:
+
+#.  `since <creation-year> by the TYPO3 contributors`,
+    for example "since 1999 by the TYPO3 contributors" (official TYPO3 manuals and TYPO3
+    system extensions)
+#.  `since <creation-year> by <vendor> & contributors`,
+    for example "since 1999 by dkd & contributors" (third-party TYPO3 extensions)
```

Comment:
```suggestion
    for example "since 1999 by John Doe & contributors" (third-party TYPO3 extensions)
```

Would like to not see real names of companies.